storage/remote/queue_manager_test.go

```diff
@@ -47,6 +47,17 @@ import (
 
 const defaultFlushDeadline = 1 * time.Minute
 
+func getHighestTimestampGaugeMetric() *maxGauge {
```

Comment:
This should be something like `newHighestTimestampMetric()`, `get` indicates that you are getting an existing metric which is incorrect.